content/Mobile_Dev/WebAuthn/IOS/Register_Only_Security_Keys.adoc

```diff
@@ -0,0 +1,120 @@
+
+=== Application demo
+Before we dive into the implementation guide, let’s first view an example of what we will be creating. The gif below demonstrates this scenario using our demo application.
+
+video::videos/reg_seckey_1.mp4[height=600]
+
+=== Invoking the WebAuthn registration ceremony
+To get started let's begin by refreshing ourselves on how to register a new credential with WebAuthn. Let's start with this basic account security screen. Here a user is given a list of their credentials, and a button that will begin the WebAuthn ceremony, allowing a user to add a new credential. In this sample the titles reflect that our application is only allowing for security keys.
+
+Take a look at Figure 1 to see an example of an account security screen for adding a credential.
+
+image::images/reg_seckey_1.jpg[width=250]
+**Figure 1**
+
+Once the user is ready to register a new credential, the “Add a new security key” button is clicked.
+
+Figure 2 shows the modal that appears on the user's screen asking if they wish to use Face ID or their security key.
+
+image::images/reg_seckey_2.jpg[width=500]
+**Figure 2**
+
+This is different from the standard modal pop-ups. Previously the user would have been asked if they wanted to use Face ID. Instead they are now only given the option to register a security key (an example of the default flow can be seen on link:/Mobile_Dev/WebAuthn/IOS/Register_Face_ID_and_Security_Keys.html[this page]).
+
+Behind the scenes, the button is triggering a call to your relying party in order to retrieve the PublicKeyCreationOptions that are used to create the new credential.
+
+Figure 3 demonstrates an example of the PublicKeyCreationOptions that were used in the ceremony above. 
+
+[source, json]
+----
+{
+  "publicKey": {
+      "rp": {
+          "name": "WebAuthn Starter Kit",
+          "id": "d2a4za4g31xyaw.amplifyapp.com/"
+      },
+      "user": {
+          "name": "ios-demo",
+          "displayName": "ios-demo",
+          "id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+      },
+      "challenge": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+      "pubKeyCredParams": [***],
+      "excludeCredentials": [
+          {
+              "type": "public-key",
+              "id": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXX-XXXXXXXXXXXXXXXXXXXXX-XXXXXXXXXXXX"
+          }
+      ],
+      "authenticatorSelection": {
+          "authenticatorAttachment": "cross-platform",
+          "residentKey": "preferred",
+          "userVerification": "preferred"
+      },
+      "attestation": "direct",
+      "extensions": {
+          "credProps": true
+      }
+  }
+}
+----
+**Figure 3**
+
+While this sample looks very similar to the options that allow for both Face ID and security keys, the primary difference can be noted in line 25. The authenticatorSelection object in the PublicKeyCreationnOptions contains a property titled authenticatorAttachment. 
```

Comment:
* When referring to actual objects in the source code I would surround those object names with backticks `. e.g. `PublicKeyCreationOptions`
* There is an extra 'n' in `PublicKeyCreationnOptions`

Reply:
Added and fixed in the latest commit

I went through the other code sample pages and added the same backticks to other code references